CONTRIBUTING.md

```diff
@@ -309,6 +309,28 @@ about:
     when someone else is asking for feedback on their code, and universe balances out.
 
 
+Backporting
+-----------
+
+Security and bug fixes can be backported from `master` to release
+branches.
+If the backport is non-trivial, it may be appropriate to open an
+additional PR, to backport the change, only after the original PR
+has been merged.
+Otherwise, backports will be done in batches by maintainers.
```

Comment:
This is intended to be a new policy?

Reply:
As far as I understood, we are just describing the commonly used, but uncodified process. Or is it something new @laanwj ?

Reply:
> This is intended to be a new policy?

It's not a new policy, just what happens [in reality (when looking at bulk backports)](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+label%3ABackport+is%3Aclosed). However if this is controversial just drop the sentence.

Reply:
Maybe mention that the `Needs backport (...)` labels are a tracking tool for maintainers and not meant as a TODO for the author of the PR.

Reply:
@laanwj see 4d47775c1

Reply:
```diff
diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
index d38e44c47..f36f29dcf 100644
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -317,7 +317,9 @@ branches.
 If the backport is non-trivial, it may be appropriate to open an
 additional PR, to backport the change, only after the original PR
 has been merged.
-Otherwise, backports will be done in batches by maintainers.
+Otherwise, backports will be done in batches by maintainers and
+the maintainers will use the proper `Needs backport (...)` labels
+when needed (the original author does not need to worry).

 A backport should contain the following metadata in the commit body:

```